src/niscope/system_tests/test_system_niscope.py

```diff
@@ -8,7 +8,15 @@
 import tempfile
 
 
-# We need a lock file so multiple tests aren't hitting the simulated HW at the same time
+# There are system tests below that need either a PXI-5124 or a PXI-5142 instead of the PXIe-5164 we use everywhere else
+# because of specific capabilities on those models. Due to internal NI bug 969274, opening a simulated session to those models
+# sometimes fails. As a workaround, the nimi-bot VMs are configured with one simulated instrument of each kind respectively
```

Comment:
I think we should qualify usages of "simulated" with the kind of simulation i.e. session-based vs. persistent.

Reply:
Fixed in #1286